### PR TITLE
[MIGRATION] note: new api migration

### PR DESCRIPTION
**Pull request by bouvyd**
- Task: https://www.odoo.com/web/#id=13294&view_type=form&model=project.task&action=333&active_id=131

### CODE REVIEW

**Comment by dbo-odoo** [comment]
Made this a few weeks ago as a bit of training.

File: addons/note/note.py [hunk 1]
```diff
@@ -20,125 +20,116 @@
...
+    @api.one
+    @api.depends('memo')
+    def _get_note_first_line(self):
+        self.name = (self.memo and html2plaintext(self.memo) or "").strip().replace('*','').split("\n")[0]
 
-        return res
+    @api.one
+    def onclick_note_is_done(self):
+        self.open = 0
+        self.date_done = datetime.now().strftime(DEFAULT_SERVER_DATETIME_FORMAT)
```
Comment (yvaucher) [inline addons/note/note.py]: You certainly want to use `fields.Datetime` helpers instead of `DEFAULT_SERVER_DATETIME_FORMAT`

Comment (dbo-odoo) [inline addons/note/note.py]: Yes, actually. Thx :+1:

Comment (dbo-odoo) [inline addons/note/note.py]: Yes, actually. Thx :+1:

**Comment by hmo-odoo** [comment]
new PR: https://github.com/odoo/odoo/pull/7280

**Pull request closed without merging**